$DragonFly: src/lib/libpcap/patches/Attic/gencode.h.patch,v 1.1 2007/03/16 12:59:10 sephe Exp $
Index: gencode.h
===================================================================
RCS file: /opt/df_cvs/src/contrib/libpcap-0.9/gencode.h,v
retrieving revision 1.1.1.1
diff -u -r1.1.1.1 gencode.h
--- gencode.h	25 Dec 2006 00:32:18 -0000	1.1.1.1
+++ gencode.h	9 Mar 2007 10:26:11 -0000
@@ -306,6 +306,8 @@
 struct block *gen_pf_action(int);
 struct block *gen_pf_dir(int);
 
+struct block *gen_p80211_type(int, int);
+
 void bpf_optimize(struct block **);
 void bpf_error(const char *, ...)
     __attribute__((noreturn, format (printf, 1, 2)));
